### DISCUSSION TITLE
Wake lock on Mobile

### DISCUSSION THREAD

**Discussion by daveytran** [korlibs/korge#851] [Q&A]
How do i request wake lock for Android/iOS in Kotlin multiplatform

**Comment by soywiz** [comment]
That functionality is not provided directly by KorGE right now. I have created a feature request here: https://github.com/korlibs/korge/issues/852

If someone knows how to implement it, please feel free to share :)

**Reply by daveytran** [reply]
Is there documentation or recommended way on how to customise the generated Android code in my project? As it is, it just overwrites on build

**Reply by soywiz** [reply]
That's a separate question. Can you open a new thread for that so it is not lost here?
